po/it/system-monitor.po

```diff
@@ -1,7 +1,7 @@
 # Traduzione italiana per il pacchetto system-monitor.
 # Copyright (C) 2012 THE system-monitor'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the system-monitor package.
-# Federico Piazza fede.piazza92@gmail.com, 2012.
```

Comment:
Differently from the Last-Translator, I would not remove this attribution. Just add yours!